drivers/CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ add_subdirectory_if_kconfig(hwinfo)
 add_subdirectory_if_kconfig(espi)
```

Comment:
maybe rework the commit title: drivers: Add LoRa driver directory

Though you could merge this with the actual driver as well. As you like.

Reply:
> maybe rework the commit title: drivers: Add LoRa driver directory

okay.

> Though you could merge this with the actual driver as well. As you like.

The reasoning behind splitting the commits is, it will be easier for other developers to look into the actual commit which adds the driver. They may use that commit as a reference for adding support for some other lora modems.

Reply:
ok